src/main/scala/rps/Main.scala

```diff
@@ -1,5 +1,28 @@
 package rps
-
+import scala.io.StdIn.readLine
+import scala.util.Random
 object Main extends App {
-  println("ciao")
+  Game.play
+}
+
+object Game {
+  def play() : Unit = {
+    val moveMap = Map("0" -> "Rock", "1" -> "Paper", "2" -> "Scissor")
+    println("Choose Rock (0), Paper(1), Scissor(2)")
+    val move = moveMap.getOrElse(readLine(),"None")
+    val computerMove = moveMap(Random.nextInt(3).toString)
+    val resultString = s"$move VS $computerMove -> "
+    val result =(move, computerMove) match {
```

Comment:
Here you could just return the gameResult String and then
`println(s"$resultString $result")`